public/css/breach-detail.css

```diff
@@ -19,23 +83,32 @@ main.breach-detail {
   flex-direction: column;
 }
 
-.priority-data-class {
-  font-weight: 600;
-}
-
-.data-action {
-  margin-left: var(--padding);
+.low-weight-data-class {
+  border: 1px solid #c689ff69;
```

Comment:
Should this be `rgba()` style?

https://caniuse.com/#feat=css-rrggbbaa vs https://caniuse.com/#feat=css3-colors